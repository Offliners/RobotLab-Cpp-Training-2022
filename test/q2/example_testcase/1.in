8
Debate
RoBE
indifFeReNt
PnEuMonoultramicRoscopiCsilicOVOlcanoconiosis
THEORETICAL
patRioT
paSSpoRt
b00k
