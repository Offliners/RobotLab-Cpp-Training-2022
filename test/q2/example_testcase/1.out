n./v. 爭論
n. 長袍
adj. 對…不感興趣
Unknown
adj. 理論(上)的
n. 愛國者
n. 護照
Unknown